backend/otone_client.py

```diff
@@ -252,25 +282,26 @@ def periodically_send_ip_addresses():
 
     subscriber = Subscriber(session_factory, loop)
     publisher = Publisher(session_factory)
-    
+
 
     while (crossbar_status == False):
         try:
-            logging.info('trying to make a connection...')
+            logger.info('trying to make a connection...')
             make_a_connection()
         except KeyboardInterrupt:
             crossbar_status = True
         except:
             #raise
             pass
         finally:
-            logging.info('error while trying to make a connection, sleeping for 5 seconds')
-            time.sleep(5)
+            if not keep_backend_running and disconnect_counter>disconnect_count_threshold:
+                # kill here
```

Comment:
If a comment describes what the code is doing than it's not useful. A developer should understand what the code does from reading it. If one wants to know what a library/framework does they should be able to do that from reading it's docs/source...

I think comments should explain why a certain approach was taken over another. Or to clarify an optimization that might not be obvious.

(I write comments like this; please call me out on it).